### DISCUSSION TITLE
DorisBE 1.2.3 Failed to initialize JNI: [RUNTIME_ERROR]Failed to find the library libjvm.so.

### DISCUSSION THREAD

**Discussion by mrg-nj** [apache/doris#18604] [A - General / Q&A]
系统:  CentOS Linux release 7.7.1908 (Core)
JDK: java version "1.8.0_202"
Java(TM) SE Runtime Environment (build 1.8.0_202-b08)
Java HotSpot(TM) 64-Bit Server VM (build 25.202-b08, mixed mode)

使用root用户
lib下存在java-udf-jar-with-dependencies.jar
其他配置均为默认

运行提示
[root@test bin]# ./start_be.sh 
WARNING: Logging before InitGoogleLogging() is written to STDERR
W0412 19:05:08.143786 19179 doris_main.cpp:382] Failed to initialize JNI: [RUNTIME_ERROR]Failed to find the library libjvm.so.

查看libjvm.so 存在 但是无法找到
/home/java/jdk1.8.0_202/jre/lib/amd64/server/libjvm.so

尝试加入 LD_LIBRARY_PATH 和 /etc/ld.so.conf   ldconfig 依旧提示  Failed to find the library libjvm.so

**Comment by yagagagaga** [comment]
可以更换 OpenJDK 试试看
